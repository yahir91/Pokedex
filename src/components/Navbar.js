import { Link } from 'react-router-dom';

const Navbar = () => (
  <div className="navbarPoke">
    <Link to="/">
      <img src="PokÃ©dex_logo.png" alt="pokedex" />
    </Link>
  </div>
);

export default Navbar;
