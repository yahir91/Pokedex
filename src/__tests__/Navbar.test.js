import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

test('Componnet renders correctly', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Navbar />
      ,
    </MemoryRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
