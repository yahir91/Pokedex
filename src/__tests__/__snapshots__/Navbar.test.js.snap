// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componnet renders correctly 1`] = `
Array [
  <div
    className="navbarPoke"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="pokedex"
        src="PokÃ©dex_logo.png"
      />
    </a>
  </div>,
  ",",
]
`;
